worldedit-core/src/main/java/com/sk89q/worldedit/function/operation/ForwardExtentCopy.java

```diff
@@ -423,8 +424,8 @@ public void cancel() {
         return ImmutableList.of(
                 TranslatableComponent.of("worldedit.operation.affected.block",
                         TextComponent.of(affectedBlocks)).color(TextColor.LIGHT_PURPLE),
-                TranslatableComponent.of("worldedit.operation.affected.biome",
-                        TextComponent.of(affectedBiomeCols)).color(TextColor.LIGHT_PURPLE),
+//                TranslatableComponent.of("worldedit.operation.affected.biome",
+//                        TextComponent.of(affectedBiomeCols)).color(TextColor.LIGHT_PURPLE),
```

Comment:
why is this commented out?

Reply:
Because it always returns 0 atm, and rather than having a wrong text I just commented it out for now

Reply:
But this might be fixed already, and will probably only cause confusion in the future, so I'm gonna comment it back in